src/services/post-install-utility.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable no-console */
```

Comment:
Please drop this line in all files

Reply:
We require this line for `displayGrid` method.

Reply:
Adding to @ravali-rimmalapudi , 
@Sindhura3  We need these lines to suppress `eslint` failures for `console` usage in code, which is desired for this case. Had removed from `welcome.js` since no console is being used in that file now.

Reply:
Ah, okay. My bad :)